src/EditorFeatures/Core.Wpf/Suggestions/PreviewChanges/PreviewChangesCodeAction.cs

```diff
@@ -30,8 +31,8 @@ public PreviewChangesCodeAction(Workspace workspace, CodeAction originalCodeActi
 
                 public override string Title => EditorFeaturesResources.Preview_changes2;
 
-                internal override async Task<ImmutableArray<CodeActionOperation>> GetOperationsCoreAsync(
-                    Solution originalSolution, IProgressTracker progressTracker, CancellationToken cancellationToken)
+                private protected override async Task<ImmutableArray<CodeActionOperation>> GetOperationsCoreAsync(
```

Comment:
>private protected  [](http://example.com/codeflow?start=16&length=18)

I thought private protected was only relevant for classes that could be derived from

Reply:
it's an override.  so it has to match the base signature (which is private-protected).